### ISSUE TITLE
[BUG] in POST v2/subscriptions if servicePath header is empty, the servicePath field is missing in DB

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#1961]
in POST v2/subscriptions if Fiware-ServicePath header is empty, the servicePath field is missing in DB, when it should be "/" value as default.

#### Subscription request
```
POST http://qa-orion-fe-02:1026/v2/subscriptions
Content-Type: application/json
Fiware-Service: test_service_path
Fiware-ServicePath:
 {"notification": {"callback": "http://localhost:1234", "attributes": ["temperture_0"]}, "expires": "2016-04-05T14:00:00.00Z", "subject": {"entities": [{"idPattern": ".*", "type": "room"}], "condition": {"attributes": ["temperature_0"]}}}
```

#### Subscription response
```
http code: 201
date: Tue, 29 Mar 2016 11:09:46 GMT
connection: Keep-Alive
content-length: 0
location: /v2/subscriptions/56fa627aeca5a9f3a56d9e58
```

#### Doc associated in mongoDB
```
{"reference": "http://localhost:1234", "throttling": 0L, "expression": {"q": "", "geometry": "", "georel": "", "coords": ""}, "format": "JSON", "entities": [{"type": "room", "id": ".*", "isPattern": "true"}], "attrs": ["temperture_0"], "_id": ObjectId("56fa627aeca5a9f3a56d9e58"), "conditions": [{"type": "ONCHANGE", "value": ["temperature_0"]}], "expiration": 1459864800L}
```

**Comment by fgalan** [comment]
What about in NGSIv1 subscriptions? They also ommit the SP in the DB in the similar case?

**Comment by iariasleon** [comment]
Yes, in NGSIv1 the behavior is the same:

#### subscription request in v1
```
http://localhost:1026/NGSI10/subscribeContext
Fiware-Service: unknownT
Fiware-ServicePath:
Accept: application/json
Content-Type: application/json
{
    "entities": [
        {
            "type": "Room",
            "isPattern": "false",
            "id": "Room1"
        }
    ],
    "attributes": [
        "temperature"
    ],
    "reference": "http://localhost:1028/accumulate",
    "duration": "P1M",
    "notifyConditions": [
        {
            "type": "ONCHANGE",
            "condValues": [
                "temperature"
            ]
        }
    ],
    "throttling": "PT5S"
}
```

#### doc stored in mongo
```
{ "_id" : ObjectId("57061e151160d8c861319055"), "expiration" : NumberLong(1462610709), "reference" : "http://localhost:1028/accumulate", "throttling" : NumberLong(5), "entities" : [ { "id" : "Room1", "type" : "Room", "isPattern" : "false" } ], "attrs" : [ "temperature" ], "status" : "active", "conditions" : [ { "type" : "ONCHANGE", "value" : [ "temperature" ] } ], "expression" : { "q" : "", "geometry" : "", "coords" : "", "georel" : "" }, "format" : "JSON" }
```

**Comment by fgalan** [comment]
Fixed in PR #2021. Please @iariasleon have a look to check if it has been finally solved and in that case close the issue.

**Comment by iariasleon** [comment]
LGTM, now servicePath is "/#"
```
 {"status": "active", "reference": "http://localhost:1234", "throttling": 0L, "expression": {"q": "", "geometry": "", "georel": "", "coords": ""}, "format": "JSON", "entities": [{"type": "room", "id": ".*", "isPattern": "true"}], "expiration": 1459864800L, "servicePath": "/#", "_id": ObjectId("570bae8d0acf2fb7aee36261"), "conditions": [{"type": "ONCHANGE", "value": ["temperature"]}], "attrs": ["temperature"]}
```
